test/worker/move-contact-worker.spec.ts

```diff
@@ -0,0 +1,132 @@
+/* eslint-disable dot-notation */
+import axios from 'axios';
+import { expect } from 'chai';
+import sinon, { SinonSandbox } from 'sinon';
+
+import Auth from '../../src/lib/authentication';
+import { moveContactQueue } from '../../src/lib/queues';
+import { 
+  MoveContactWorker, 
+  MoveContactData, 
+  JobResult 
+} from '../../src/worker/move-contact-worker';
+
+describe('worker/move-contact-worker', () => {
+  let worker: MoveContactWorker;
+  let sandbox: SinonSandbox;
+  const queueName = 'testQueue';
+  const jobData: MoveContactData = {
+    parentId: 'parent123',
+    contactId: 'contact123',
+    sessionToken: 'token123',
+    instanceUrl: 'http://test-instance.com'
+  };
+  const job = {
+    id: 'job123',
+    name: 'testJob',
+    data: jobData,
+    opts: {},
+    log: sinon.stub(),
+    moveToDelayed: sinon.stub(),
+  };
+
+  let shouldPostponeStub: sinon.SinonStub;
+  let moveContactStub: sinon.SinonStub;
+  let postponeStub: sinon.SinonStub;
+  let decodeTokenStub: sinon.SinonStub;
+
+  beforeEach(() => {
```

Comment:
not required in this PR

i think a valuable test would be an integration test which calls `MoveLib.move()`, mocks the interactions with BullMQ (or which actually hosts redis during the test), and then verifies the `cht move-contacts` command which is kicked off by the runner

i love this sort of integration test and find it really value. the unit-style testing you have here (particularly tests focusing on private interfaces) are nice, but very sensitive to implementation change.  some integration tests or tests which are closer to user expectations would be great